src/s390x/mod.rs

```diff
@@ -20,3 +22,10 @@ pub use zipl::chreipl;
 pub use zipl::zipl;
 mod eckd;
 mod fba;
+
+#[derive(ArgEnum, Clone, Debug)]
+pub enum ZiplMode {
+    Auto,
+    EnforceSecex,
+    DisableSecex,
+}
```

Comment:
And then we'd rename this to `ZiplSecexMode` and the members would be `Auto`, `Enforce`, and `Disable`.